src/Quartz/Impl/AdoJobStore/StdAdoConstants.cs

```diff
@@ -33,317 +35,318 @@ public class StdAdoConstants : AdoConstants
         public const string TablePrefixSubst = "{0}";
 
         // table prefix substitution string
+        [Obsolete("Scheduler name injection has been replaced with sql parameter @schedulerName")]
         public const string SchedulerNameSubst = "{1}";
```

Comment:
I've replaced anywhere this was originally used with @schedulerName, and left this for an external uses. 
I'm hoping this will avoid breaking changes to external code.